tests/tracking/fluent/test_fluent.py

```diff
@@ -204,38 +181,39 @@ def _is_function_or_class(obj):
     assert set(apis).issubset(set(mlflow.__all__))
 
 
-def test_get_experiment_id_from_env():
+def test_get_experiment_id_from_env(monkeypatch):
     # When no env variables are set
-    HelperEnv.assert_values(None, None)
+    assert not MLFLOW_EXPERIMENT_NAME.is_defined
+    assert not MLFLOW_EXPERIMENT_ID.is_defined
     assert _get_experiment_id_from_env() is None
 
     # set only ID
     name = "random experiment %d" % random.randint(1, 1e6)
     exp_id = mlflow.create_experiment(name)
     assert exp_id is not None
-    HelperEnv.set_values(experiment_id=exp_id)
-    HelperEnv.assert_values(exp_id, None)
+    monkeypatch.delenv(MLFLOW_EXPERIMENT_NAME.name, raising=False)
+    monkeypatch.setenv(MLFLOW_EXPERIMENT_ID.name, exp_id)
     assert _get_experiment_id_from_env() == exp_id
 
     # set only name
     name = "random experiment %d" % random.randint(1, 1e6)
     exp_id = mlflow.create_experiment(name)
     assert exp_id is not None
-    HelperEnv.set_values(name=name)
-    HelperEnv.assert_values(None, name)
+    monkeypatch.delenv(MLFLOW_EXPERIMENT_ID.name, raising=False)
+    monkeypatch.setenv(MLFLOW_EXPERIMENT_NAME.name, name)
     assert _get_experiment_id_from_env() == exp_id
 
     # create experiment from env name
     name = "random experiment %d" % random.randint(1, 1e6)
-    HelperEnv.set_values(name=name)
-    HelperEnv.assert_values(None, name)
+    monkeypatch.delenv(MLFLOW_EXPERIMENT_ID.name, raising=False)
+    monkeypatch.setenv(MLFLOW_EXPERIMENT_NAME.name, name)
     assert MlflowClient().get_experiment_by_name(name) is None
     assert _get_experiment_id_from_env() is not None
```

Comment:
This block looks strange. The comment says create experiment, but it doesn't.

Reply:
by setting an experiment name that doesn't exist yet, `_get_experiment_id_from_env()` will create a new experiment of that name

Reply:
Makes sense!